ReadMe needs updating

1. meerdag Makefile script no longer exists
```sh
~/github.com/qitmeer/qng (main) $ make meerdag
make: *** No rule to make target `meerdag'.  Stop.
```
2. meerevm
The existing scripts are:
```sh
~ git clone github.com/Qitmeer/meerevm # missing https://
~ cd ./cmd/evm # no cmd directory
~ go build
```

which would supposed to be:
```sh
~ git clone https://github.com/Qitmeer/meerevm
~ cd ./evm
~ go build
```

however, after my correction, the build is failing:

```sh
 $ go build
# gopkg.in/olebedev/go-duktape.v3
In file included from _cgo_export.c:4:
debugger.go:23:13: warning: unused function '_duk_debugger_attach' [-Wunused-function]


```

--- a/README.md
+++ b/README.md
@@ -3,19 +3,6 @@ The next generation of the Qitmeer network implementation with the plug-able VMs
 
 ## qng
 ```bash
-~ make qng
+~ make
 ```
 
-## meerdag
-```bash
-~ make meerdag
-```
-
-## meerevm
-```bash
-~ git clone github.com/Qitmeer/meerevm
-~ cd ./cmd/evm
-~ go build
-```
-
-* If you want to use this module `meerevm`, you can copy it to QNG's plug-in directory.(Default directory:build/bin/plugin)

